### ISSUE TITLE
keyStream for all keys

### ISSUE THREAD

**Issue by emilbayes** [dominictarr/level-sublevel#99]
Currently I can't stream all keys in my leveldb due to what is happening (I assume) around here: [nut.iterator#L168](https://github.com/dominictarr/level-sublevel/blob/389e1329a5492daf9e4e53ef86f92779d2a56394/nut.js#L168).

Can this be "undone" in some way? "Fixing" is a bug would warrant a breaking change and a major version bump. I encountered this key when working on a custom `keyEncoding`, which I couldn't get to work. I'll send a test if I can narrow it down :)

**Comment by emilbayes** [comment]
I guess this is related #63 and #25

**Comment by dominictarr** [comment]
this is the intended design, however, if you keep a handle to the raw database
``` js
var _db = level(path)
var db = sublevel(db)
_db.createReadStream() // <-- this will have everything!
```

**Comment by emilbayes** [comment]
Arr alright, is that because you can have a sublevel with the empty string as name?

**Comment by dominictarr** [comment]
no, it's just that partitioning the database into subdatabases is the idea. it's meant to be contained. breaking out of the containment is not a feature I want to add to sublevel, but it is something you can do, via the method I describe.
